### PR TITLE
Ben expand inactivitymessages

### PR DESCRIPTION
**Pull request by benlower**

### CODE REVIEW

File: docs/src/content/docs/api/calls.mdx [hunk 1]
```diff
@@ -537,6 +542,83 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+  </div>
+  <div>
+    <dt class="font-mono">endBehavior</dt>
+    <dd class="ml-4"><span class="text-gray-600">string</span> - The behavior to exhibit when the message is finished being spoken. Must be one of the enumerated values.</dd>
+    <dd class="ml-4"><span class="text-gray-600">enum</span></dd>
+    <dd class="ml-8">
+      <ul class="list-disc space-y-2">
+        <li><code>END_BEHAVIOR_UNSPECIFIED</code> - Default. The system will continue to wait for user input.</li>
+        <li><code>END_BEHAVIOR_HANG_UP_SOFT</code> - Will end the call unless the user interacts while the agent is delivering the message.</li>
+        <li><code>END_BEHAVIOR_HANG_UP_STRICT</code> - Immediately terminates the call after speaking the message, without waiting for any additional user interaction.</li>
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:582]:
```suggestion
        <li><code>END_BEHAVIOR_HANG_UP_STRICT</code> - Will end the call after speaking the message, regardless of whether the user interrupts.</li>
```

File: docs/src/content/docs/api/calls.mdx [hunk 2]
```diff
@@ -537,6 +542,83 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+      "message": "Hello, are you still there?"
+    },
+    {
+      "duration": "10s",
+      "message": "If there's nothing else, may I end the call?"
+    },
+    {
+      "duration": "5s",
+      "message": "Thank you for calling. Have a great day. Goodbye.",
+      "endBehavior": "END_BEHAVIOR_HANG_UP_STRICT"
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:605]: These durations are too tight for an example. Also non-strict hangups are probably preferable in most cases.

```suggestion
      "duration": "30s",
      "message": "Are you still there?"
    },
    {
      "duration": "30s",
      "message": "If there's nothing else, may I end the call?"
    },
    {
      "duration": "45s",
      "message": "Thank you for calling. Have a great day. Goodbye.",
      "endBehavior": "END_BEHAVIOR_HANG_UP_SOFT"
```

File: docs/src/content/docs/api/calls.mdx [hunk 3]
```diff
@@ -537,6 +542,83 @@ Call recordings are only generated if you add `"recordingEnabled": true` to the
...
+}
+```
+
+Here's what would happen based on the example above:
+
+<Steps>
+1. Call starts.
+1. After 5 seconds of no user interaction, agent says "Hello, are you still there?".
+1. If user interacts, call continues. If no user interaction occurs for 10 seconds, agent says "If there's nothing else, may I end the call?".
+1. If no user interaction occurs for another 5 seconds, agent says the provided message and the call ends.
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:617]:
```suggestion
1. Call starts.
1. After 30 seconds of no user interaction, agent says "Are you still there?".
1. If user interacts, call continues. If no user interaction occurs for another 30 seconds, agent says "If there's nothing else, may I end the call?".
1. If no user interaction occurs for another 45 seconds, agent says the provided message and the call ends unless the agent is interrupted during this final message.
```

File: docs/src/content/docs/api/calls.mdx [hunk 4]
```diff
@@ -561,10 +643,14 @@ Here's an example:
 ```
 
 #### Using Mistral
-If you are using `fixie-ai/ultravox-mistral-nemo-12B` as your model, you need to do the following when creating the call:
-1. **Empty System Prompt** → Set `systemPrompt` to an empty string.
-1. **Prompt in Initial Messages** → Add a single user message to `initialMessages` that contains the system prompt.
-1. **Proper Turns** → Make sure that you follow Mistral's strict, alternating, user > agent > user message ordering.
+:::caution[Important Note]
+Mistral uses a simplified prompt structure.
```
Comment (mdepinet) [inline docs/src/content/docs/api/calls.mdx:647]:
```suggestion
Mistral requires a simplified prompt structure.
```

**Review by mdepinet — APPROVED** [review]

File: docs/src/content/docs/api/calls.mdx (hunk 2 above)
Comment (benlower) [reply docs/src/content/docs/api/calls.mdx:605]: committing this but will edit durations a bit

**Pull request merged**